docs/docs/behaviors/key-toggle.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Key Toggle Behavior
+sidebar_label: Key Toggle
+---
+
+## Summary
+
+The key toggle behavior toggles the press of a key.
+If the key is not currently pressed, key toggle will press it, holding it until the key toggle is pressed again or the key is released in some other way.
+If the key _is_ currently pressed, key toggle will release it.
+Example uses for key toggle include shift lock, or `ALT-TAB` window switching without holding down the `ALT` modifier.
```

Comment:
I'm not sure if these are intended to be new paragraphs, but if so a blank line is required between each of them. 

I do think the last line (with the examples) would be useful broken out into its own paragraph.

Reply:
These were not meant to be separate paragraphs; I just usually put each sentence on a new line for easier to read diffs when changes are made.  Agreed on that last line, made that its own paragraph.